opnode/test/system_test.go

```diff
@@ -299,6 +299,8 @@ func TestSystemE2E(t *testing.T) {
 	seqBlock, err := l2Seq.BlockByNumber(context.Background(), receipt.BlockNumber)
 	require.Nil(t, err)
 	require.Equal(t, verifBlock.Hash(), seqBlock.Hash(), "Verifier and sequencer blocks not the same after including a batch tx")
+
+	time.Sleep(1 * time.Minute)
```

Comment:
will remove this before merge. Useful to keep it open a bit longer to see everything is healthy.